pkg/runtime/pubsub/subscriptions.go

```diff
@@ -199,7 +199,8 @@ func GetSubscriptionsGRPC(channel runtimev1pb.AppCallbackClient, log logger.Logg
 				s, ok := status.FromError(rErr)
 				if ok && s != nil {
 					if s.Code() == codes.Unimplemented {
-						return rResp, backoff.Permanent(rErr)
+						log.Infof("pubsub subscriptions: gRPC app does not implement ListTopicSubscriptions")
```

Comment:
```suggestion
						log.Warnf("pubsub subscriptions: gRPC app does not implement ListTopicSubscriptions")
```

Reply:
Why warning? Apps are not required or expected to implement this. It means a hello world app would get this warning - which is why I am using Info.